app/assets/stylesheets/application.scss

```diff
@@ -340,7 +348,7 @@ table {
 .spinner-placeholder {
   width: 16px;
   height: 16px;
-  background: image-url('spinner.gif');
+  background: url('spinner.gif');
```

Comment:
We would need to rebase indeed, spinner_placeholder function was modified, this css class is no longer relevant.